base/gcutils.jl

```diff
@@ -259,4 +259,13 @@ function enable_logging(on::Bool=true)
     ccall(:jl_enable_gc_logging, Cvoid, (Cint,), on)
 end
 
+"""
+    GC.is_logging()
+
+When turned on, print statistics about each GC to stderr.
```

Comment:
This sounds like the docstring for enable logging, not querying that state

Reply:
oops, fixed now